cmd/commands/cli/command_identities.go

```diff
@@ -310,3 +320,85 @@ func (c *cliApp) setBeneficiary(actionArgs []string) {
 		}
 	}
 }
+
+const usageExportIdentity = "export <identity> <new_passphrase> [file]"
+
+func (c *cliApp) exportIdentity(actionsArgs []string) {
+	dataDir := c.config.GetStringByFlag(config.FlagDataDir)
+	if dataDir == "" {
+		clio.Error("Could not get data directory")
+		return
+	}
+
+	if len(actionsArgs) < 2 || len(actionsArgs) > 3 {
+		clio.Info("Usage: " + usageExportIdentity)
+		return
+	}
+	id := actionsArgs[0]
+	passphrase := actionsArgs[1]
+
+	ksdir := node.GetOptionsDirectoryKeystore(dataDir)
+	ks := keystore.NewKeyStore(ksdir, keystore.LightScryptN, keystore.LightScryptP)
+
+	ex := identity.NewExporter(identity.NewKeystoreFilesystem(ksdir, ks))
+
+	blob, err := ex.Export(id, "", passphrase)
+	if err != nil {
+		clio.Error("Failed to export identity: ", err)
+		return
+	}
+
+	if len(actionsArgs) == 3 {
+		filepath := actionsArgs[2]
+		write := func() error {
+			f, err := os.Create(filepath)
+			if err != nil {
+				return err
+			}
+			defer f.Close()
+
+			_, err = f.Write(blob)
+			return err
+		}
+
+		err := write()
+		if err == nil {
+			clio.Success("Identity exported to file:", filepath)
+			return
+		}
+
+		clio.Warn("Export succeeded, file write failed. Will print to screen.")
```

Comment:
This `Will print to screen` could be a problem. If the user does not expect private keys to be printed to stdout we should not do it. If something failed the user can do a second attempt or print to stdout manually.